platform/darwin/src/http_file_source.mm

```diff
@@ -133,8 +132,8 @@ void cancel() {
 
     // Avoid %s here because it inserts hidden bidirectional markers on macOS when the system
     // language is set to a right-to-left language.
-    [userAgentComponents addObject:[NSString stringWithFormat:@"MapboxGL/%@ (%@)",
-                                    CFSTR(MBGL_VERSION_STRING), CFSTR(MBGL_VERSION_REV)]];
+    [userAgentComponents addObject:[NSString stringWithFormat:@"MapboxGL/0.0.0 (%@)",
```

Comment:
This is a reasonable change: other than requiring a version to be here, we don’t check its value anywhere as far as I know. (Another option would be to hard-code it to the last version that it was set to, even if that version isn’t particularly meaningful. That would preserve sort order for this part of the string, but the SDK version appears before this component anyways.)